libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -406,23 +406,21 @@ void Aircraft::set_speedup(float speedup)
 /*
   Create and set in/out socket
 */
-void Aircraft::set_interface_ports(const char* address, const int port_in, const int port_out)
+void Aircraft::set_interface_ports(const char* local_address, const char* gazebo_address, const int port_in, const int port_out)
```

Comment:
this should be const char* gazebo_address but simulator_address to be generic, as ideal the set_interface_ports should be use by every SITL plugin